app/src/main/kotlin/me/tomasan7/jecnamobile/student/StudentProfileRepository.kt

```diff
@@ -0,0 +1,10 @@
+package me.tomasan7.jecnamobile.student
+
+import io.github.tomhula.jecnaapi.data.student.Locker
+import io.github.tomhula.jecnaapi.data.student.Student
+
+interface StudentProfileRepository
+{
```

Comment:
A fourth test comment to verify https://youtrack.jetbrains.com/issue/IJPL-224686